### DISCUSSION TITLE
What is the release plan for Metalama

### DISCUSSION THREAD

**Discussion by steinerm55** [metalama/Metalama#41] [Q&A] [answered]
Is there already a more or less reliable release plan for Metalama?

I have to start new projects and I can use the current prerelease version, but I could not use it for creating production releases.

Some milestones could be helpful.

Thanks

**Comment by gfraiteur** [comment, marked as answer]
The release plan is that we are feature-complete and almost, but not yet, code-complete (i.e. there are still some minor gaps) and we are now working on stabilizing the product. When we will have a feeling that the product is stable, we will make it Release Candidate, then General Availability. How long this will still take before the last milestone, we don't know.

Since there is no run-time component in Metalama, it does not make sense to avoid using it for a production release if you use if for development anyway. However I understand you may be bound by corporate policies that would make sense for run-time components, but actually apply to all components you use.

So unfortunately I cannot give you a short answer.

**Comment by steinerm55** [comment]
Thank you for the quick response.

I will discuss the matter internally.

Regards
